app/controllers/dit_landing_page_controller.rb

```diff
@@ -1,3 +1,31 @@
 class DitLandingPageController < ApplicationController
+  around_action :switch_locale
+
   def show; end
+
+  helper_method :translation_links
```

Comment:
Functions in this controller certainly need tests, but I wanted to check the approach before committing to it.

Reply:
Once we merge, and run [this](https://github.com/alphagov/collections-publisher/pull/1196) I believe the content item will contain the available translations in the [available translations array](https://www.gov.uk/api/content/eubusiness) so we won't need to hardcode them in the controller. ie we can fetch them from the content item and pass them to the translation nav component. But in terms of roll out, my feeling is that hardcoding is fine for now. We can open a refactor PR after this one is merged and live.  If you added a controller test eg

```
it "renders the page for the es locale" do
  get :show, params: { locale: "es" }
  assert_response :success
  assert_select "h1", "Comercio con el Reino Unido a partir del 1 de enero de 2021 como empresa con sede en la UE"
end
```

That would cover us for now, rather than extensively testing the `translation links` method which will be removed.